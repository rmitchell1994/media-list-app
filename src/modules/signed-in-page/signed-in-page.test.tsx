import React from "react";
import { render } from "../../../test/test-utils";
import SignedInPage from "./signed-in-page";

describe("Signed In Page", () => {
  it("should render component", () => {
    const { asFragment } = render(<SignedInPage />);

    expect(asFragment()).toMatchSnapshot();
  });
});
