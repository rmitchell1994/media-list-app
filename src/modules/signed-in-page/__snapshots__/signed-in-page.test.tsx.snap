// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Signed In Page should render component 1`] = `
<DocumentFragment>
  <div>
    Signed In Page
  </div>
</DocumentFragment>
`;
